assets/scss/_variables_project.scss

```diff
@@ -90,3 +90,20 @@ h2, h3 {
 }
 
 
+.workflow {
```

Comment:
We tried a couple alternatives here, and felt that the titles weren't critical, and lead to a weirder design and didn't make it more readable. Since this is coming up again we should revisit - although I don't think it should be a blocker for this PR. In the meantime we can also refer to the condition --> task --> resolution flow in the intro for the workflows.

Reply:
Cool cool - as long as it was intentional and pros and cons were considered, we're good.